package.json

```diff
@@ -34,7 +34,6 @@
     "@types/react": "^17.0.3",
     "@types/react-dom": "^17.0.3",
     "@types/react-router-dom": "^5.1.7",
-    "@types/uuid": "^8.3.0",
```

Comment:
The `uuid` type wasn't doing what I hoped it would do, so I removed it.  I wanted to be able to define a `uuid` type on the IBeacons interface so we could enforce that IDs are actual UUIDs.  This didn't work out, as a UUID string (e.g. `"1fd77a0a-db47-4708-9600-d95c3a4bf7a4"`) can't be easily typecast to a `typeof UUID`.  I've replaced the `uuid` data type with plain old `string`.